### PR TITLE
Adding styled-component + styled-reset in oackages and a line in fron…

### PR DESCRIPTION
**Pull request by julianecasier**
…tend/index with a script to add fontawesome.

### CODE REVIEW

**Comment by SDecambray** [comment]
Okey

**Review by ghost — DISMISSED** [review]

File: frontend/index.html [hunk 1]
```diff
@@ -9,5 +9,9 @@
   <body>
     <div id="root"></div>
     <script type="module" src="/src/main.jsx"></script>
+    <script
+      src="https://kit.fontawesome.com/f831de03a8.js"
```
Comment (LuigiMenez) [inline frontend/index.html]: Not good way import with React

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
Not good Way import with react

**Review by ghost — DISMISSED** [review]

**Review by LuigiMenez — DISMISSED** [review]
Ok

**Review by ghost — DISMISSED** [review]
okey

**Review by LuigiMenez — DISMISSED** [review]
Ok for me

File: package.json [hunk 2]
```diff
@@ -17,5 +17,10 @@
   "devDependencies": {
     "concurrently": "^7.0.0",
     "husky": "^7.0.0"
+  },
+  "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^6.1.1",
+    "@fortawesome/free-solid-svg-icons": "^6.1.1",
+    "@fortawesome/react-fontawesome": "^0.1.18"
```
Comment (LoicBrassart) [inline package.json]: Nope, you don't need these packages in the root package.json, they should be in the `frontend` one (React needs it, your backend doesn't)

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
A bit of cleanup to do ;-)

**Comment by LoicBrassart** [comment]
@LuigiMenez Beware of *which* `package.json` holds the new installations, it can lead to serious bugs

**Review by ghost — APPROVED** [review]

**Review by LuigiMenez — APPROVED** [review]
Ok for me

**Review by LoicBrassart — APPROVED** [review]
No more Evils !
You don't use FAwesome in the end ?

**Comment by julianecasier** [comment]
@LoicBrassart  No, too much time spent on it :-)

**Pull request merged**